modules/freetype/src/freetype.cpp

```diff
@@ -723,10 +723,28 @@ int FreeType2Impl::coFn( const FT_Vector *cnt,
 
     PathUserData *p = (PathUserData *)user;
 
+    // Calicurate upper limit of CtoL.
+    // It is same as total length of polylines{ mOldP, cnt, to }.
+
+    const double dx1 = p->mOldP.x  -  cnt->x;
+    const double dy1 = p->mOldP.y  -  cnt->y;
+    const double dx2 = cnt->x      -  to->x;
+    const double dy2 = cnt->y      -  to->y;
+
+    const int maxL = static_cast<int>(
+        std::pow( dx1 * dx1 + dy1 * dy1, 0.5 ) +
+        std::pow( dx2 * dx2 + dy2 * dy2, 0.5 )
```

Comment:
pow 0.5 is sqrt. I propose to use sqrt as it's faster and more obvious.

Reply:
Thank you for your review! Yes, it is better, I fixed it.